src/nnops.jl

```diff
@@ -93,3 +93,17 @@ end
 function apply(normalization::UniformAdditiveNormalization, X::LazySet)
     return translate(X, fill(normalization.shift, dim(X)))
 end
+
+#### Solver using the CH of the sampled outputs as an inner approx of the real output
+@with_kw struct SampledApprox <: Solver
+    nsamples::Int64 = 10000
+end
+
+function NeuralVerification.forward_network(solver::SampledApprox, nnet, input)
+    samples = sample(input, solver.nsamples)
+    outputs = Vector()
```

Comment:
under the 1D assumption this is just a `Vector{Float64}(undef, solver.nsamples)`.